src/screens/Game.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useCallback, useState } from "react";
+import {
+  StyleSheet,
+  Button,
+  View,
+  SafeAreaView,
+  Text,
+  Alert,
+} from "react-native";
+
+import { getGame } from "../utils/api";
+
+const GameScreen = ({ navigation, route }) => {
+  const [game, setGame] = useState();
+
+  const handleGetGame = useCallback(
```

Comment:
I'm not sure that the use of this `useCallback` hook is justified here! 

> This is useful when passing callbacks to optimized child components that rely on reference equality to prevent unnecessary renders

I have a feeling that there will be a better way to make your asynchronous call in your `useEffect`.